packages/services/src/emerald-client/ApiAccess.ts

```diff
@@ -121,7 +114,7 @@ export class EmeraldApiAccess {
   }
 
   public newPricesListener(): PriceListener {
-    return new PriceListener(this.pricesClient);
+    return new PriceListener(this.marketClient);
```

Comment:
I don't understand this change, why the Prices Listener is supposed to use the Markets Client instead of a Prices Client?